cli_tools/gce_ovf_export/exporter/instance_export_preparer.go

```diff
@@ -0,0 +1,113 @@
+//  Copyright 2020 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package ovfexporter
+
+import (
+	"context"
+	"fmt"
+
+	daisyutils "github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/common/utils/daisy"
+	ovfexportdomain "github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/gce_ovf_export/domain"
+	"github.com/GoogleCloudPlatform/compute-image-tools/daisy"
+	"google.golang.org/api/compute/v1"
+)
+
+type instanceExportPreparerImpl struct {
```

Comment:
would you add intro for responsibilities of cleaner, preaprer, etc?

Reply:
Interfaces that these classes implement, as well as NewCleaner/Preparer/... constructors have these docs. Let me know if you thing more is needed.